File: MotivApp/app/src/main/java/com/motiv/piotr/PhotosFragmentContract.java
package com.motiv.piotr;

import androidx.fragment.app.*;
import dagger.*;
import dagger.android.*;
import dagger.android.support.*;
import javax.inject.*;

public interface PhotosFragmentContract {

    public interface View {

        public void photosPagerAdaptersetData(java.util.List<com.motiv.piotr.Photo> arg0);
    }

    public interface Presenter {

        public void goRestApigetPhotosList();
    }
}

File: MotivApp/app/src/main/java/com/motiv/piotr/PhotosFragmentContract.kt
 
package com.motiv.piotr
import kotlin.collections.List

public interface PhotosFragmentContract {

    public interface View {

        public fun photosPagerAdaptersetData(arg0: List<com.motiv.piotr.Photo>): Unit
    }

    public interface Presenter {

        public fun goRestApigetPhotosList(): Unit
    }
}
